app/frontend/schools/settings/components/SchoolCustomize__Root.res

```diff
@@ -489,7 +489,7 @@ let make = (~authenticityToken, ~customizations, ~schoolName, ~schoolAbout) => {
       </div>
       <div className="mt-6 font-bold"> {t("icon")->str} </div>
       <div className="mt-3 w-1/2 max-w-sm">
-        <div className="bg-gray-300 rounded-t-lg h-12 flex items-end">
+        <div className="bg-gray-300 rounded-t-lg h-12 flex items-end icon-display">
```

Comment:
@irajsuhail Was the addition of this class name intentional? No styles were found here.